Rules/CommonScripts/plants_regrow.as

```diff
@@ -0,0 +1,199 @@
+#define SERVER_ONLY
+
+const bool moss_stone = false; //should stone turn into mossy stone
+
+const f32 grass_grow_chance = 10;
+const f32 bush_grow_chance = 1;
+const f32 moss_stone_chance = 2;

```

Comment:
It looks like these variables can range from 0-100 rather than 0-1. Why? It makes more sense if chance / probability / percentages range from 0-1.